tests/pages/remoteInstancesPage.js

```diff
@@ -294,4 +297,16 @@ module.exports = {
     I.waitForVisible(this.fields.requiredFieldHostname, 30);
     I.waitForVisible(this.fields.requiredFieldPort, 30);
   },
+
+  createPostgreSQLInstance(serviceName, tracking) {
```

Comment:
Done we have the same [here](https://github.com/percona/pmm-ui-tests/blob/042260fa61d9c9e7b52f6396706c9459c4ec7de9/tests/pages/remoteInstancesPage.js#L164-L170)?

Reply:
@yurkovychv In this method, we cannot specify different service names. Do you want me to refactor the old method? :)

Reply:
As I see it should work [this](https://github.com/percona/pmm-ui-tests/blob/PMM-6748_Add_Instance_page/tests/pages/remoteInstancesPage.js#L205-L206) way?